atomate/vasp/workflows/presets/core.py

```diff
@@ -201,18 +202,33 @@ def wf_elastic_constant(structure, c=None):
     c = c or {}
     vasp_cmd = c.get("VASP_CMD", VASP_CMD)
     db_file = c.get("DB_FILE", DB_FILE)
-    # @montoyj - any comments?
+    # @montoyj - any comments on the following being configurable params?
```

Comment:
Yeah, I'll remove these in my next PR, which basically has a single wf for the "MP" elastic procedure and a "minimal" one as well.